src/handlers/shared/get-user-role.ts

```diff
@@ -0,0 +1,28 @@
+import { Context } from "../../types";
+
+export async function getUserRole(context: Context, user: string): Promise<string> {
+  const orgLogin = context.payload.organization?.login;
+
+  if (!orgLogin) {
+    throw new Error("Organization login not found in context payload.");
+  }
+
+  try {
+    const response = await context.octokit.orgs.getMembershipForUser({
+      org: orgLogin,
+      username: user,
+    });
+
+    const role = response.data.role;
+
+    if (role === "admin") {
```

Comment:
Why don't you just return the role?

Reply:
at second glance it looks so dumb😅! I've modified it now

Reply:
I'm sure I received the exact same advice from someone early on in the team too, I have a clear memory of it but can't find the comment 😂